### DISCUSSION TITLE
About obfuscator on nuxt3

### DISCUSSION THREAD

**Discussion by kurodaak1ra** [nuxt/nuxt#15953] [Questions]
how to package with obfuscator, i found all package from npm about vite obfuscator,  but no one can use, i need some help, thanks

this is rollup-obfuscator
<img width="1066" alt="スクリーンショット 2022-09-02 8 25 25" src="https://user-images.githubusercontent.com/38561128/188034275-172e844d-4be4-4a51-a3da-3f0f05439c18.png">

vite.config.js
<img width="264" alt="スクリーンショット 2022-09-02 8 29 47" src="https://user-images.githubusercontent.com/38561128/188034379-6071645f-db83-462c-b8c6-619ff10ae7ff.png">

**Comment by mokurofuzo** [comment]
Have you solved it

**Reply by kurodaak1ra** [reply]
No, but you can use vite minify, like this
<img width="152" alt="スクリーンショット 2023-01-12 17 11 15" src="https://user-images.githubusercontent.com/38561128/212026104-9f965a8f-0766-4246-8b80-ddac709eafdc.png">

**Reply by mokurofuzo** [reply]
let me try it
